src/rustup.rs

```diff
@@ -39,7 +39,7 @@ fn rustup_command(msg_info: &mut MessageInfo, no_flags: bool) -> Command {
         Verbosity::Quiet => {
             cmd.arg("--quiet");
         }
-        Verbosity::Verbose => {
+        Verbosity::Verbose(2..) => {
```

Comment:
Is this the desired behavior? I'd expect all subcommands would print verbose output if I ran `cross --verbose` or something similar.

Reply:
this was basically the whole point of the pr, my expectation for `cross --verbose` is to only print the command invocations and show their outputs. Like `cargo -v`, who wont print all `rustc` invocations, but will with `-vv`